src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -129,19 +132,68 @@ public String getOgcCollectionFeatures(
 							content=@Content())
 			},
 			externalDocs=@ExternalDocumentation(url="http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_feature_")
-			)
+		)
 	@GetMapping(value = "collections/{collectionId}/items/{featureId}", produces = MediaType.APPLICATION_JSON_VALUE)
 	public String getOgcCollectionFeature(
 			@RequestParam(value = "f", required = false, defaultValue = "json") String mimeType,
 			@PathVariable(value = "collectionId") String collectionId,
 			@PathVariable(value = "featureId") String featureId, HttpServletResponse response) {
 
-		String rtn = collectionsDao.getCollectionFeatureJson(collectionsParams.buildParams(collectionId, featureId));
-		if (rtn == null) {
-			response.setStatus(HttpStatus.NOT_FOUND.value());
-		}
+		// verify the collection exist before fetching the feature
+		getOgcCollection(mimeType, collectionId, response);
+		return doIfResponseOk(response, 
+				()->collectionsDao.getCollectionFeatureJson(collectionsParams.buildParams(collectionId, featureId)));
+	}
 
```

Comment:
Since we are not distinguishing between the collection not being found and the feature not being the
found, just the dao should just be called. It will return null if the collection id does not exists.

Reply:
I removed this for another reason anyhow.